src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
               <h1 data-cy="title">
-                {{team.name}}
-                <ion-button data-cy-role="editModal" v-if="canEdit" color="light" @click="intendEditTitle" fill="clear">
-                  <ion-icon size="small" :icon="editIcon" />
-                </ion-button>
+                {{ team.name }}
+                <ion-icon
+                  size="small"
+                  :icon="editIcon"
+                  data-cy-role="editModal"
+                  v-if="canEdit"
+                  color="light"
+                  @click="intendEditTitle"
+                />
               </h1>
+
               <inline-link-list :items="socialLinks" showIcon>
-                <li v-if="canEdit">
-                  <ion-button @click="intendEditSocialLink" color="light" size="small" fill="clear">
-                    <ion-icon size="small" :icon="editIcon" />
-                  </ion-button>
-                </li>
+                <div v-if="canEdit" style="display: inline">
+                  <ion-icon
+                    size="small"
+                    :icon="editIcon"
+                    @click="intendEditSocialLink"
+                    color="light"
+                  />
+                </div>
               </inline-link-list>
+
+              <div class="extra-actions" v-if="canEdit">
+                <ion-chip
+                  title="remove background"
+                  v-if="team.background"
+                  @click="removeBackground"
+                  outline
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="trashIcon" />
+                </ion-chip>
+
+                <ion-chip
+                  v-else
+                  @click="selectBackground"
+                  title="upload background"
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="uploadIcon" />
+                </ion-chip>
+                <inline-link-list showTitle :items="footerLinks">
+                  <li v-if="canEdit">
+                    <ion-button
+                      @click="intendEditFooterLinks"
+                      size="small"
+                      fill="clear"
+                    >
+                      <ion-icon size="small" :icon="editIcon" />
+                    </ion-button>
+                  </li>
+                </inline-link-list>
+              </div>
+            </ion-col>
+          </div>
+
+          <ion-toolbar>
+            <!-- Scrollable Segment -->
+            <ion-segment
+              scrollable
+              value="about"
+              mode="md"
+              @ionChange="segmentChanged($event)"
+            >
+              <ion-segment-button value="qrcode">
+                <ion-icon :icon="qrCodeIcon" />
+              </ion-segment-button>
+              <ion-segment-button value="about">
+                <ion-label>About</ion-label>
```

Comment:
Let's make them i18n, too, please!
```suggestion
                <ion-label>{{ $('team.tabs.about') }}</ion-label>
```

and following of course...